kinder/third_party/kind/loadbalancer/config.go

```diff
@@ -64,6 +64,9 @@ backend kube-apiservers
 // Config returns a kubeadm config generated from config data, in particular
 // the kubernetes version
 func Config(data *ConfigData) (config string, err error) {
+	if 0 > data.ControlPlanePort {
```

Comment:
hi, there is no way for kinder to expose the port configuration.
why do you think this check is needed?

also it should be written as `data.ControlPlanePort <= 0`